mix.exs

```diff
@@ -64,7 +64,9 @@ defmodule DarkWorldsServer.MixProject do
       {:cors_plug, "~> 3.0"},
       {:tesla, "~> 1.4", override: true},
       {:toxiproxy_ex, "~> 1.0.0"},
-      {:uuid, "~> 1.1"}
+      {:uuid, "~> 1.1"},
+      {:eflambe, "~> 0.3.0"},
```

Comment:
Why add `eflambe`?

Reply:
I did not find a way to save the raw results of flame_on, but eflambe does have that option if we ever want it, i thought it would be useful, should I remove it?

Reply:
No idea about the need to save them, but if you think is useful good by me